packages/@okta/vuepress-site/docs/guides/embedded-siw/main/index.md

```diff
@@ -391,11 +391,11 @@ See the following:
 
 * Android:
 * [Sign users in to your Android mobile app using the redirect model](/docs/guides/sign-into-mobile-app-redirect/android/main/)
-* [Browser Sign In](https://github.com/okta/samples-android/tree/master/browser-sign-in)
+* [Browser sign-in](https://github.com/okta/samples-android/tree/master/browser-sign-in)
```

Comment:
Actually, neither are right as we don't use sign-in as a noun. It would need to be Browser sign-in flow

Reply:
Changed to browser sign-in flow